assets/src/design-system/components/textArea/stories/index.js

```diff
@@ -0,0 +1,184 @@
+            onChange={handleChange}
+            label={text('TextArea 1 Label', 'Normal')}
+            hint={text('Hint', 'Hint')}
+            placeholder="placeholder"
+          />
+          <TextArea
+            aria-label="input-two"
+            id="two-light"
+            name="twoLight"
+            value={inputState.twoLight}
+            onChange={handleChange}
+            label={text('TextArea 2 Label', 'Error')}
+            hint={text('Hint', 'Hint')}
+            placeholder="placeholder"
+            hasError
+          />
+          <TextArea
+            aria-label="disabled-input-one"
+            id="three-light"
+            name="threeLight"
+            value={inputState.threeLight}
+            onChange={handleChange}
+            label={text('TextArea 4 Label', 'Disabled')}
+            hint={text('Hint', 'Hint')}
+            placeholder="placeholder"
+            disabled
+          />
+          <TextArea
+            aria-label="input-four"
+            id="four-light"
+            name="fourLight"
+            value={inputState.fourLight}
+            onChange={handleChange}
+            label={text('TextArea 4 Label', 'Normal')}
+            hint={text('Hint', 'Hint')}
+            placeholder="placeholder"
+            showCount={true}
+            maxLength={20}
+          />
+        </Row>
+      </Container>
+      <DarkThemeProvider>
+        <Container darkMode>
+          <Row>
+            <TextArea
+              aria-label="input-four"
+              id="one-dark"
+              name="oneDark"
+              value={inputState.oneDark}
+              onChange={handleChange}
+              label={text('TextArea 1 Label', 'Normal')}
+              hint={text('Hint', 'Hint')}
+              placeholder="placeholder"
+            />
+            <TextArea
+              aria-label="input-five"
+              id="two-dark"
+              name="twoDark"
+              value={inputState.twoDark}
+              onChange={handleChange}
+              label={text('TextArea 2 Label', 'Error')}
+              hint={text('Hint', 'Hint')}
+              placeholder="placeholder"
+              hasError
+            />
+            <TextArea
+              aria-label="disabled-input-two"
+              id="three-dark"
+              name="threeDark"
+              value={inputState.threeDark}
+              onChange={handleChange}
+              label={text('TextArea 3 Label', 'Disabled')}
+              hint={text('Hint', 'Hint')}
+              placeholder="placeholder"
+              disabled
+            />
+            <TextArea
+              aria-label="input-four"
+              id="four-dark"
+              name="fourDark"
+              value={inputState.fourDark}
+              onChange={handleChange}
+              label={text('TextArea 4 Label', 'Normal')}
+              hint={text('Hint', 'Hint')}
+              placeholder="placeholder"
+              showCount={true}
```

Comment:
Just a small nit: we don't need to pass `true` here. Not important though 😅 
```suggestion
              showCount
```